crypto/x509/pcy_node.c

```diff
@@ -113,6 +116,13 @@ X509_POLICY_NODE *ossl_policy_level_add_node(X509_POLICY_LEVEL *level,
     return node;
 
  node_error:
+    if (level != NULL && node_stored) {
```

Comment:
hmm, `if (node_stored)` would be enough, since it implies `level != NULL`

Reply:
Instead of playing chichen-and-eggs with the compiler, I would prefer a new jump label with a descriptive
name here.
Then we would not need the partially redundant `node_stored` variable at all.

Reply:
@neverpanic please note that node can be also set in level->anyPolicy. Can you please set it to NULL if `level->anyPolicy == node`?

Reply:
I think the code already did that, but the next push will hopefully make this clearer by using a different condition and a separate jump label.